media/videostitcher/create_live_session.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2023 Google LLC
```

Comment:
Presubmit is asking you to update the copyright year on these since it is now 2024 (sorry about that 😦 ). This also needs to be updated to submit.

Reply:
Done